Frontend/src/components/Input/PopupFormular.svelte

```diff
@@ -130,6 +130,7 @@
                                     <DateInput
                                         disabled={input.disabled}
                                         bind:timeMillis={$keyValueStore[input.bindTo.keyValueStoreKey][input.bindTo.attr]}
+                                        quickset={'quickset' in input ? input.quickset : {}}
```

Comment:
`input.quickset ?? {}`
would be shorter

Reply:
Ah thanks, that works! Nice, Python could use something like that.